fjord/base/google_utils.py

```diff
@@ -31,6 +31,9 @@ def ga_track_event(params, async=True):
        testing/development data hosing your production data.
 
     """
+    if sys.argv and sys.argv[0].endswith('py.test'):
```

Comment:
Yeah... The other way to do it is like what we do with gengo where we only have the real credentials on production and the other environments don't have them. Then if you don't have them, then it no workey.

However, GA is slightly different in that our id is both in a js file and in python. So we could do a setting, but we'd need to do it in a way that it's obvious it only affects the python code or something like that.

I'm not sure what other options there are.